playbook.md

```diff
@@ -827,31 +819,92 @@ TODO. This section to cover:
+## Working arrangements
+### Time
+Our working hours are 10:00-18:00, Monday to Friday. Some people work different hours by arrangement. Anyone is free to do that as long as their hours of work don't make it hard for other people to get things done. For example, many people arrive earlier than 10:00 and leave earlier, which is generally fine.
 
-During each meeting, we add cards to the [buddy trello](https://trello.com/b/TmiYBPZg/buddy-chats), so that the rest of the team know what you're working on and can help. If you want to talk about something private, you can. You don't have to put everything on the board. But the presumption is that we do, unless there's a good reason not to.
+We have a short stand-up every morning, where we each tell the whole team about a single thing we will get done that day. The stand-up is at 10:00. If you miss the stand-up, then you are [late](#lateness).
 
-The buddy system is there to help us all improve, to make sure we're doing the right kind of work and to check that we're dealing with practical problems we're facing. It's not line management, so it's not the right place to talk about holiday, salary, sickness, absence or anything disciplinary. If you need to talk about those things, talk to Vee or Harry.
+Most developers have maintenance responsibilities, which they do during [ticket time](#ticket-time).
 
-There are a two main ideas that underpin this system.
+We do our very best to work to a [sustainable pace](#sustainable-pace). But sometimes, when we're approaching a firm deadline or a launch, or a client is having an emergency, we work longer hours than normal. From time to time, there's an emergency that means we have to work during unsociable hours to solve the problem. Neither of these happen very often, but they are a normal part of life at dxw.
 
-#### We are each responsible for our own improvement
-Before this system, we had regular performance reviews. They were supposed to be quarterly but generally ended up being annual. They didn't work very well, and were very top-down. We don't think this is the right way.
+### Holiday
 
-We prefer to assume that everyone wants to improve, and will do so given the time and space to work on it. The buddy system is there to provide a set of prompts to ensure that this happens, and an explicit time to talk about it.
 
-#### It's best to be open
-Trying to improve things in secret is a missed opportunity to get help from others. We may each be responsible for our own improvement, but we're also a team: we should help each other. Recording the things we're working on with the buddy Trello is an unintrusive, quick way to ensure that everyone else knows too.
+### Sickness
+If you are sick, you must let us know as soon by 11:00, or as soon as is reasonably practical. Do not come to the office. If you come to the office when sick, you may be sent home again.
```

Comment:
I'm not sure 11am is early enough to tell someone they are sick, unless it's serious? By serious I mean an emergency or if you're unable to actually contact someone. I would expect to know before standups with clients so I can inform them and make arrangements if I need to.